logging.go

```diff
@@ -101,42 +84,22 @@ func appScopeLogger(
 	buildVersion string,
 ) *Logger {
 	entry := logger.WithFields(logrus.Fields{
-		"app":           appName,
+		"application":   appName,
 		"env":           envName,
 		"build_version": buildVersion,
 		"scope":         "app",
 	})
 	return &Logger{entry}
 }
 
-func requestScopeLogger(
-	logger *logrus.Logger,
-	r *http.Request,
-	route string,
-	ip string,
-	correlationID string,
-	appName string,
-	envName string,
-	buildVersion string,
-) *Logger {
+func requestScopeLogger(logger *logrus.Logger, ip string, correlationID string, appName string, envName string, buildVersion string) *Logger {
 	ctxLogger := logger.WithFields(logrus.Fields{
-		"app":            appName,
+		"application":    appName,
 		"env":            envName,
 		"build_version":  buildVersion,
 		"scope":          "request",
 		"correlation_id": correlationID,
-		"url":            r.RequestURI,
```

Comment:
Ive removed this since I dont think we should be logging headers and url ect for every log statement in the request context. 
The request logger will enable linking this up with the correlation ID so we can filter out the stream and see then headers ect.